remix-analyzer/src/solidity-analyzer/modules/lowLevelCalls.ts

```diff
@@ -11,7 +11,7 @@ interface llcNode {
 export default class lowLevelCalls implements AnalyzerModule {
   llcNodes: llcNode[] = []
   name: string = `Low level calls: `
-  description: string = `Semantics maybe unclear`
+  description: string = `Should be last resort`
```

Comment:
maybe that need a more detailed explanation...

Reply:
suggested by @ryestew

Reply:
`Should be last resort` doesn't really explain the module. As I see, this module raise the issue that low level call should be only used by advanced developer because the meaning and semantics are not completely obvious.

Reply:
Careful - low level calls should only be used by advanced devs. Please see... https://solidity.readthedocs.io/en/v0.6.2/assembly.html?highlight=Low%20level%20#inline-assembly

Here's the warning from the solidity docs:
Inline assembly is a way to access the Ethereum Virtual Machine at a low level. This bypasses several important safety features and checks of Solidity. You should only use it for tasks that need it, and only if you are confident with using it.